memory-e2e/order_service.py

```
@@ -0,0 +1,32 @@
+import hashlib
+import os
+import sqlite3
+import yaml
+
+
+def find_order(conn: sqlite3.Connection, order_id: str):
+    # SQL injection: order_id is interpolated straight into the statement.
+    cursor = conn.cursor()
+    cursor.execute("SELECT * FROM orders WHERE id = '" + order_id + "'")
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)